src/shared/compatibility.js

```diff
@@ -101,14 +101,17 @@ const hasDOM = typeof window === 'object' && typeof document === 'object';
   if (div.classList.toggle('test', 0) === false) {
     return;
   }
-  const originalDOMTokenListToggle = DOMTokenList.prototype.toggle;
 
   DOMTokenList.prototype.toggle = function(token) {
     if (arguments.length > 1) {
       const force = !!arguments[1];
       return (this[force ? 'add' : 'remove'](token), force);
     }
-    return originalDOMTokenListToggle(token);
+
+    if (this.contains(token)) {
+      return (this.remove(token), false);
+    }
+    return (this.add(token), true);
```

Comment:
Arguable, but would it have been work calling the original function?  The main benefit would be that if the underlying implementation to `toggle` changes, then this never needs to change.  i.e.:

```js
return originalDOMTokenListToggle.call(this, token);
```

Reply:
Regardless, thanks for this fix!

Reply:
I think that's essentially what we had before this fix and it didn't work in IE. Since this is only a polyfill for IE and toggle functionality is trivial, I don't think we need to change this.

Reply:
The original call was this:

```js
return originalDOMTokenListToggle(token);
```

My suggestion, which also fixes the problem in IE, is this:

```js
return originalDOMTokenListToggle.call(this, token);
```

It's ultimately not a big deal since the issue is now fixed.

Reply:
Ah, I see. The diff in this patch is actually old; in the current code it's minimized to https://github.com/mozilla/pdf.js/blob/master/src/shared/compatibility.js#L110-L113. In the current code I don't think calling the original function is really possible anymore since that one doesn't support the `force` parameter, so then we would need to special-case it and get back to the code in this diff. Indeed it doesn't really matter too much since it's a working polyfill and it will probably be removed once we drop IE support in a later major release anyway.

Reply:
Seems like a moot point then, thanks for calling that out!